shared/src/validation-schemas/clinic/clinic.validation-schema.ts

```diff
@@ -16,10 +16,8 @@ const addClinic = yup.object().shape({
     .mixed<ClinicType>()
     .oneOf(Object.values(ClinicType))
     .required(),
-//   [ClinicKey.IMAGE_PATH]: yup
-//     .string()
-//     .required(RegisterValidationMessage.IMAGE_REQUIRED)
-//     .url(RegisterValidationMessage.IMAGE_INCORRECT),
+  [ClinicKey.CITY_ID]: yup
+    .string(),
```

Comment:
```suggestion
    .string(),
    .url(RegisterValidationMessage.IMAGE_INCORRECT)
```
Remove it if we don't use it `RegisterValidationMessage.IMAGE_REQUIRED`

Reply:
Missed